### PR TITLE
[MINOR][SparkR] R API documentation for "coltypes" is confusing

### PR DESCRIPTION
**Pull request by keypointt**
## What changes were proposed in this pull request?

R API documentation for "coltypes" is confusing, found when working on another ticket.

Current version http://spark.apache.org/docs/2.0.0/api/R/coltypes.html, where parameters have 2 "x" which is a duplicate, and also the example is not very clear

current

screen shot 2016-08-03 at 5 56 00 pm
## How was this patch tested?

Tested manually on local machine. And the screenshots are like below:

screen shot 2016-08-07 at 11 29 20 pm

screen shot 2016-08-03 at 5 56 22 pm

### CODE REVIEW

**Comment by keypointt** [comment]
hi @shivaram do you mind have a look at this minor patch? thank you very much :)

File: R/pkg/R/DataFrame.R [hunk 1]
```diff
@@ -41,7 +41,7 @@ setOldClass("structType")
 #'\dontrun{
 #' sparkR.session()
 #' df <- createDataFrame(faithful)
-#'}
+#' }
```
Comment (shivaram) [inline R/pkg/R/DataFrame.R]: Do we need the change of adding space before `}` in this PR ?

Comment (keypointt) [inline R/pkg/R/DataFrame.R]: I'm not sure, when I scan through the R source files, I see both 'with one-space' and 'with no-space'.

so I should revert the adding space ones?

Comment (shivaram) [inline R/pkg/R/DataFrame.R]: Yeah lets not do them in this PR. If we want to fix this lets do them in a separate PR ?

Comment (keypointt) [inline R/pkg/R/DataFrame.R]: sure I'll revert it :)

File: R/pkg/R/DataFrame.R [hunk 2]
```diff
@@ -411,7 +411,7 @@ setMethod("coltypes",
 #'
 #' Set the column types of a SparkDataFrame.
 #'
-#' @param x A SparkDataFrame
+#' @param y A SparkDataFrame whose column types to be set
```
Comment (felixcheung) [inline R/pkg/R/DataFrame.R]: is there a reason you change this from x to y?

Comment (keypointt) [inline R/pkg/R/DataFrame.R]: to avoid duplicate variables, now it has 2 'x' in 'ARGUMENTS' section, or is there a better way to do it?

http://spark.apache.org/docs/2.0.0/api/R/coltypes.html

https://cloud.githubusercontent.com/assets/3925641/17386808/effb98ce-59a2-11e6-9657-d477d258a80c.png

Comment (felixcheung) [inline R/pkg/R/DataFrame.R]: I see - it's common in R to get the getter/setter method to share the same document page, and event parameter lists, eg.
https://stat.ethz.ch/R-manual/R-devel/library/base/html/colnames.html

I think the best way to do that is not to rename it to `y`, but leave out the `@param x` for `coltype<-`

Comment (keypointt) [inline R/pkg/R/DataFrame.R]: ok I'll try to do it, thanks Felix

**Comment by felixcheung** [comment]
LGTM

**Comment by keypointt** [comment]
@shivaram

**Comment by shivaram** [comment]
LGTM. @felixcheung feel free to merge this and let me know if the commit scripts work fine

**Comment by shivaram** [comment]
@felixcheung FYI I'm also backporting this to branch-2.0 as it gets rid of one of the CRAN warnings.

**Comment by keypointt** [comment]
hi @shivaram what is the warning message? I can fix it here

**Comment by shivaram** [comment]
The warning I got in `branch-2.0` before this PR was 
```
Duplicated \argument entries in documentation object 'coltypes':
  ‘x’
```
With the backport this warning goes away, so nothing needs to be fixed.

**Comment by keypointt** [comment]
got it, thanks @shivaram
